clock/index.js

```diff
@@ -23,6 +23,7 @@
  * clockJobConfig object in config.js.
  */
 const conf = require('../config');
+const logger = require('../logger');
```

Comment:
Need to init the producer here as well, since this is an entry point for a separate process.